taipy/gui/_gui_cli.py

```diff
@@ -81,21 +81,55 @@ class _GuiCLI(_AbstractCLI):
         "--no-reloader": {"dest": "taipy_no_reloader", "help": "No reload on code changes", "action": "store_true"},
     }
 
+    __BROWSER_ARGS: Dict[str, Dict] = {
+        "--run-browser": {
+            "dest": "taipy_run_browser",
+            "help": "Open a new tab in the system browser",
+            "action": "store_true",
+        },
+        "--no-run-browser": {
+            "dest": "taipy_no_run_browser",
```

Comment:
Actually after some thought I know why it was made this way: we want the CLI to have the most weight at the end of the day.
If you use a CLI option, it will win.
Now we don't have a way to detect whether an option has been mentioned or not, except setting some value to True.
Indeed that means that for Boolean values, we have to separated options ON and OFF, each being potentially set to True so that we don't enforce anything if no option was set.